config/types/root_types.ts

```diff
@@ -228,6 +228,28 @@ export const parseOrganization = (raw: any): Organization => {
   };
 };
 
+export const generatePageURLForUnifiedDocument = (
```

Comment:
We have a function called `getUrlToUniDoc` in routing.js which does this

Reply:
Ah gotchya, updated to use that one instead.